terraform/modules/gcp/vpc/main.tf

```diff
@@ -1,57 +1,33 @@
-locals {
-  prefix = var.prefix
-}
-
-# VPC
-resource "google_compute_network" "vpc" {
-  name                    = var.vpc_name
+resource "google_compute_network" "vpc_network" {
```

Comment:
Shouldn't we use something like https://github.com/terraform-google-modules/terraform-google-network ? Like we do on AWS?

I believe this is official google module.